commerce-dashboard-web/src/main/java/com/liferay/commerce/dashboard/web/internal/constants/CommerceDashboardPortletKeys.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.commerce.dashboard.web.internal.constants;
+
+/**
+ * @author Andrea Di Giorgi
+ * @author Alessio Antonio Rendina
```

Comment:
You should really be listed as the author here (and on these other files), @gcmznt :)